flagging_site/blueprints/flagging.py

```diff
@@ -112,4 +112,33 @@ def output_model() -> str:
             in reach_model_mapping.items()
         }
 
-    return render_template('output_model.html', tables=reach_html_tables)
\ No newline at end of file
+    return render_template('output_model.html', tables=reach_html_tables)
+
+@bp.route('/flags')
+def flags() -> str:
+
+    df = get_data()
+
+    flags_1 = {
+        'Newton Yacht Club': reach_2_model(df, rows=1)['safe'].iloc[0],
+        'Watertown Yacht Club': reach_2_model(df, rows=1)['safe'].iloc[0],
+        'Community Rowing, Inc.': reach_2_model(df, rows=1)['safe'].iloc[0],
+        'Northeastern\'s Henderson Boathouse': reach_2_model(df, rows=1)['safe'].iloc[0]
+    }
+ 
+    flags_2 = {
+        'Paddle Boston at Herter Park': reach_2_model(df, rows=1)['safe'].iloc[0],
+        'Harvard\'s Weld Boathouse': reach_3_model(df, rows=1)['safe'].iloc[0],
+        'Riverside Boat Club': reach_4_model(df, rows=1)['safe'].iloc[0],
+        'Charles River Yacht Club': reach_5_model(df, rows=1)['safe'].iloc[0],
+    }
+
+    flags_3 = {
+        'Union Boat Club': reach_5_model(df, rows=1)['safe'].iloc[0],
+        'Community Boating': reach_5_model(df, rows=1)['safe'].iloc[0],
+        'Paddle Boston at Kendall Square': reach_5_model(df, rows=1)['safe'].iloc[0]
+    }
+
+    flags = [flags_1, flags_2, flags_3]
```

Comment:
Ah I see what you're doing here. I would actually suggest doing the following instead: just output all the flags normally, and make sure the flags just "wrap around" when the page size is shrunk.

If we want to do max 4 per row, then what we'd do is make it so the iframe's max width results in a max of 4 per row.

The reason we don't want to hardcode these line breaks in is because we want the page to render smoothly on smaller screens. E.g. if the screen is smaller and someone's screen width only accommodates 3 per row, then we don't want it to be 3 - 1 - 3 - 1 - 3. We'd rather it be 3 - 3 - 3 - 2.